src/reactviews/pages/QueryResult/table/plugins/headerFilter.plugin.ts

```diff
@@ -238,6 +376,86 @@ export class HeaderFilter<T extends Slick.SlickData> {
         this.setFocusToColumn(columnDef);
     }
 
+    private async createFilterList(): Promise<void> {
```

Comment:
Does this list need to be virtualized?

Reply:
@cssuh Can you test filtering with something like 10k different rows?